<launch>

    <!-- Tracker node -->
    <node pkg="blort_ros" type="gltracker_node" name="blort_tracker" args="$(find blort_ros)">
        <param name="launch_mode" value="tracking" />
        <remap from="/blort_camera_info" to="/disparity_segmentation/camera_info" />
        <remap from="/blort_image" to="/disparity_segmentation/image_masked" /> 
    </node>
    
    <!-- Detector node -->
    <node pkg="blort_ros" type="gldetector_node" name="blort_detector" args="$(find blort_ros)">
        <remap from="/blort_camera_info" to="/disparity_segmentation/camera_info" />
    </node>

</launch>

